uikit/positionlockfab/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <declare-styleable name="Theme">
+        <attr name="positionLockFabStyle" format="reference"/>
+    </declare-styleable>
+
+    <declare-styleable name="PositionLockFab">
+        <attr name="android:tint" format="reference|color"/>
```

Comment:
to ako som ti posielal aj ten link, tam by malo byt iba `name`, format nie, lebo to moze byt, ze to prepises povodnu definiciu. malo by stacit iba `<attr name="android:tint" />`

Reply:
normalne by som dokonca cakal, ze by sa stazoval na redefiniciu...

Reply:
pravdu máš ;)